src/theme.js

```diff
@@ -1,5 +1,5 @@
 export const theme = {
-  color: {
+  colors: {
```

Comment:
tutaj przy zmianie nazw nam się posypią style w komponentach :D

Reply:
dzięki że to zauważyłeś, wycofałem to zmianę , brekponints też zmieniłem na liczbę pojedynczą